frontend/src/global_styles/content/_project_overview.sass

```diff
@@ -55,6 +55,8 @@
     &:last-child
       margin-bottom: 10px
 
+.op-enterprise-banner
```

Comment:
I have two issues with that rule: 

1. Since the file is a global style, it adds a margin to **all** banners. It is not scoped in any way.
2. It might be picky, but the projectOverview is a completely different page than the one you want to tackle. The projectOverview is the first menu item inside a project, not the admin -> project settings. 

So please, check how the spacing is handled on other places. I assume, that the html-structure differs somehow. If that is not the case, add a modifier to that specific banner and handle it in the correct sass file.